model_weight: "/home/roboticslab/Storage/YoloTag/Yolo_prev_weights/train8/weights/best.pt"
outlier_dist: 3.0
alpha: 0.96
conf_thrs: 0.45

Tags:
  "3": [0.81, 7.5, 1.28]
  "62":  [1.49, 7.5, 1.28]
  "2": [2.11, 7.5, 1.28]
  "63":  [2.71, 7.5, 1.28]
  "7":   [3.31, 7.5, 1.28]
  "67":  [3.93, 7.5, 1.28]
  "4":   [4.53, 7.5, 1.28]
  "64":  [5.16, 7.5, 1.28]


camera_matrix: [537.292878, 0.000000, 427.331854, 0.000000, 527.000348, 240.226888, 0.000000, 0.000000, 1.000000]
dist_coeffs: [0.004974, -0.000130, -0.001212, 0.002192, 0.000000]